domain/config/registrationTypes.ts

```diff
@@ -4,14 +4,15 @@ export const registrationTypes: RegistrationType[] = [
   {
     value: 'avco/ecvo',
     label: `ACVO/ECVO Diplomate/Board Eligible*`,
-    price: 450,
+    price: 550,
     optionalComment: '*actively taking boards at the moment'
   },
-  { value: 'speaker', label: 'Speaker', price: 350 },
+  { value: 'speaker', label: 'Speaker', price: 450 },
   {
     value: 'resident/inter',
     label: 'Resident/Ophtalmology Intern',
-    price: 250
+    price: 300
   },
-  { value: 'others', label: 'Others', price: 550 }
+  { value: 'others', label: 'Other Veterinary Ophthalmologists', price: 650 },
+  { value: 'guest', label: 'Guest entry for reception', price: 75 }
```

Comment:
Let’s remove this one for now. It shouldn’t deduct from the total allowed registrations.